File: api/v2010/dependent_phone_number/read-dependent-pns-example-1/read-dependent-pns-example-1.5.x.cs
// Install the C# / .NET helper library from twilio.com/docs/csharp/install

using System;
using Twilio;
using Twilio.Rest.Api.V2010.Account.Address;


class Program 
{
    static void Main(string[] args)
    {
        // Find your Account Sid and Token at twilio.com/console
        // DANGER! This is insecure. See http://twil.io/secure
        const string accountSid = "ACXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX";
        const string authToken = "your_auth_token";

        TwilioClient.Init(accountSid, authToken);

        var dependentPhoneNumbers = DependentPhoneNumberResource.Read(
            pathAddressSid: "AD2a0747eba6abf96b7e3c3ff0b4530f6e"
        );

        foreach(var record in dependentPhoneNumbers)
        {
           Console.WriteLine(record.Sid);
        }
    }
}

File: api/v2010/dependent_phone_number/read-dependent-pns-example-1/read-dependent-pns-example-1.5.x.rb
# Download the helper library from https://www.twilio.com/docs/ruby/install
require 'rubygems'
require 'twilio-ruby'

# Your Account Sid and Auth Token from twilio.com/console
# DANGER! This is insecure. See http://twil.io/secure
account_sid = 'ACXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX'
auth_token = 'your_auth_token'
@client = Twilio::REST::Client.new(account_sid, auth_token)

dependent_phone_numbers = @client
  .addresses('AD2a0747eba6abf96b7e3c3ff0b4530f6e')
  .dependent_phone_numbers
  .list

dependent_phone_numbers.each do |record|
  puts record.sid
end
